BeagleBone_black/src/data/data.hpp

```diff
@@ -124,6 +124,15 @@ struct Motors {
   int32_t rpm_BR;
 };
 
+// -----------------------------------------------------------------------------
+// Communications data
+// -----------------------------------------------------------------------------
+
+struct Communications {
+  uint32_t stopCommand;
+  uint32_t killPowerCommand;
+};
```

Comment:
Is there a reason for these to be `uint32_t`? To me they look like `bool`. (If there is a reason, don't change it. I genuinely don't know and am just suggesting.)